packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -75,10 +75,31 @@ export function createClassFeaturePlugin({
 
           if (path.isPrivate()) {
             const { name } = path.node.key.id;
+            const getName = `get_${name}`;
```

Comment:
Could you use a space instead of _? Otherwise this code throws:
```js
class A {
  get #priv() {}
  #get_priv() {}
}
```

Reply:
Also, it should throw for this code:
```js
class A {
  get #priv() {}
  #priv() {}
}
```

Reply:
Good point, will update.

Reply:
It appears that `toIdentifier` [replaces whitespaces in names with dashes](https://github.com/babel/babel/blob/3f9a1c08cc70ac5bfc21bff13ef51eea1bd06f27/packages/babel-types/src/converters/toIdentifier.js#L8) and then [strips the dash out altogether](https://github.com/babel/babel/blob/3f9a1c08cc70ac5bfc21bff13ef51eea1bd06f27/packages/babel-types/src/converters/toIdentifier.js#L14), so the end result is `getMethodName` and `setMethodName`...

Reply:
Well, `setName` and `getName` are only used in the `privateNames` set and never passed to `toIdentifier`, right?

Anyway, to fix both the problems I think you could use the `privateNames` set alongside with a `partialAccessors` map. `partialAccessor` would store the private name as a key, and the value would either be `"get"` or `"set"`.

The check to disallow private names would then look like this:
```js
if (privateNames.has(name)) {
  if ((kind === "get" && partialAccessors.get(name) === "set") || (kind === "set" && partialAccessors.get(name) === "get")) {
    partialAccessors.delete(name);
  } else {
    throw;
  }
} else {
  if (isAccessor) partialAccessors.set(name, kind);
  privateNames.add(name);
}
```

If you have a better idea feel free to ignore me!

Reply:
Alternatively, I could do: 

```js
if (prop.node.kind === "get") {
  update.getId = t.identifier(`get ${prop.scope.generateUid(`${name}`)}`);
} else if (prop.node.kind === "set") {
  update.setId = t.identifier(`set ${prop.scope.generateUid(`${name}`)}`);
}
```

...but that throws since:

```js
/* ... */
get: get _privateFieldValue2,
/* ... */
```

is a `SyntaxError` (unexpected identifier).

Reply:
The problem is only in the `privateNames` map. The `_get_privateField` identifiers are ok since they are already unique.

Reply:
@nicolo-ribaudo Ok, this should be fixed now!

Reply:
It is ok